Content.Server/Flash/FlashSystem.cs

```diff
@@ -123,6 +139,30 @@ public void Flash(EntityUid target, EntityUid? user, EntityUid? used, float flas
             _stun.TrySlowdown(target, TimeSpan.FromSeconds(flashDuration/1000f), true,
                 slowTo, slowTo);
 
+            //For Rev conversion (This probably shouldn't go in each and every flash but I'm honestly not sure where or how to put this somewhere else.)
```

Comment:
what you should do here is introduce a RevolutionaryConversionFlash, and have the flash fire a new event (let's call it `PostFlashActionEvent`) that contains details about the flashed user.

Reply:
Honestly, I tried to take a crack at it but I'm not really sure how I am supposed to do this